src/invoices/InvoiceDetails/InvoiceDetailsContainer.js

```diff
@@ -510,6 +538,7 @@ InvoiceDetailsContainer.manifest = Object.freeze({
     accumulate: true,
     fetch: false,
   },
+  orders: ordersResource,
```

Comment:
don't use mutators in new code

Reply:
In an ideal world, I'd agree with this statement, @usavkov-epam, but I think the only good options here are:

1. keep things on this page consistent and add the existing `ordersResource`, a small piece of work
2. completely refactor the entire component, including multiple things in `src/common/resources`, to use `react-query`, a large piece of work

This isn't my repo so it's not my call, but personally I'm OK with option 1.